.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,43 @@
+# Github action definitions for unit-tests with PRs.
+
+name: tfma-unit-tests
+on:
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+  workflow_dispatch:
+
+jobs:
+  unit-tests:
+    if: github.actor != 'copybara-service[bot]'
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        # Would like to add 3.11, but it causes errors
```

Comment:
```suggestion
```

Please open an issue for 3.11 support.

Reply:
[Done](https://github.com/tensorflow/model-analysis/issues/185)

Reply:
Could you also remove the comment, since we're now tracking the issue on github?

Reply:
Done